src/structures/fittedeva/maximumlikelihoodeva.jl

```diff
@@ -286,3 +286,29 @@ function transform(fm::MaximumLikelihoodEVA{ThresholdExceedance})
     return MaximumLikelihoodEVA(model, θ̂)
 
 end
+
+
+"""
+    cint(fm::MaximumLikelihoodEVA, clevel::Real=.95)::Array{Array{Float64,1},1}
+
+Compute the Wald parameter confidence intervals using the approximate parameter estimates covariance matrix.
+"""
+function cint(fm::MaximumLikelihoodEVA, clevel::Real=.95)::Array{Array{Float64,1},1}
```

Comment:
Now that we have the confidence interval for the parameters directly, should I change the tests to use this function when testing estimated parameter values?

Reply:
Éventuellement ce serait effectivement une bonne façon de faire.